packages/dotcom-ui-app-context/src/client/loadAppContext.ts

```diff
@@ -1,18 +1,7 @@
+import { loadClientEmbed } from '@financial-times/dotcom-ui-client-embed/src/client'
```

Comment:
This `src/client` business seems incorrect, I need to figure out the entry points to this package.